src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/keybinding-emacs-uncompressed.js

```diff
@@ -638,9 +638,8 @@ var screenToTextBlockCoordinates = function(x, y) {
     var col = Math.floor(
         (x + this.scrollLeft - canvasPos.left - this.$padding) / this.characterWidth
     );
-    var row = Math.floor(
-        (y + this.scrollTop - canvasPos.top) / this.lineHeight
-    );
+
+    var row = (y + this.scrollTop - canvasPos.top) / this.lineHeight;
 
     return this.session.screenToDocumentPosition(row, col);
```

Comment:
It should be okay. Note that this logic mirrors what Ace does in the default `screenToTextCoordinats()` function here: https://github.com/ajaxorg/ace/blob/f757c8568fba3e59392383e332519e5570f54e1e/lib/ace/virtual_renderer.js#L1422-L1424